package.json

```diff
@@ -11,14 +11,15 @@
   "author": "Nicholas C. Zakas",
   "main": "lib/index.js",
   "scripts": {
-    "lint": "eslint lib/ tests/",
-    "lint:fix": "eslint lib/ tests/ --fix",
+    "lint": "eslint lib/ tests/ tools/",
+    "lint:fix": "eslint lib/ tests/ tools/ --fix",
     "docs": "eslint-docs",
     "docs:check": "eslint-docs check",
     "format-no-write": "prettier-eslint lib/**/*.js tests/**/*.js --eslint-config-path=.eslintrc --eslint-ignore --prettier-ignore --eslint-path=node_modules/eslint --config=.prettierrc",
```

Comment:
I missed this - add `tests/**/*.js` to this script too

Reply:
@bradzacher ohh i forgot about this one to, 

added